internal/i18n/locales/de/default.po

```diff
@@ -102,6 +102,12 @@ msgstr "SMS Textnachricht (empfohlen)"
 msgid "codes.issue.sms-text-message-label"
 msgstr "Patienten Telefonnummer"
 
+msgid "codes.issue.sms-template-label"
+msgstr "SMS Vorlage"
+
+msgid "codes.issue.sms-template-detail"
+msgstr "Der Patient erhält eine SMS mit der ausgewählten Nachrichtenvorlage."
```

Comment:
I don't know enough german to refute that `ausgewählten Nachrichtenvorlage` is `selected template` but it sure looks long

Reply:
@hostirosti could you sanity check these two new German translations please?

Reply:
LGTM